runtime/tutor.txt

```diff
@@ -416,6 +415,296 @@ _________________________________________________________________
+ --> ... sentence doesn't have it's first and last ... .
+ --> ... sentence doesn't have it's first and last ... .
+     This sentence doesn't have it's first and last word.
+
+=================================================================
+=                        CHAPTER 4 RECAP                        =
+=================================================================
+
+ * Type y to yank (copy) text and p to paste.
+   * Type space-Y and space-P to yank/paste on the system
+     clipboard.
+
+ * Type ~ to alternate case of selected letters.
+   * Use ` and alt-` to set the case of selected layers to
+     upper and lower respectively.
+
+ * Type Q to record and stop recording a macro.
+   * Type q to repeat the recorded macro.
+
+
+
+
+
+
+
+
+=================================================================
+=                    5.1 USING THE JUMPLIST                     =
+=================================================================
+
+ Helix can keep track of "jumps" which are big movements, like
+ jumping to the definition of a function in code. It stores
+ these in what's called the jumplist.
+
+ Type C-s (ctrl-s) to manually save your current position to
+ the jumplist.
+
+ Type C-i and C-o to move forward and backwards in the jumplist
+ respectively.
+
+ 1. Type C-s somewhere.
+ 2. Move far away in the file.
+ 3. Type C-o (just once!) to come back to where you saved.
+
+
+
+
+
+=================================================================
+=                     5.2 SEARCHING IN FILE                     =
+=================================================================
+
+ Type / to search forward in file, enter to confirm search.
+ Type n to go to the next search match.
+ Type N to go to the previous search match.
+
+ 1. Type / and type in a common word, like 'banana'.
+ 2. Type enter to confirm the search.
+ 3. Use n and N to cycle through the matches.
+
+ Like the select command, searching also uses regex.
+
+ Note: To search backwards, type ? (shift-/).
+
+ Note: Unlike Vim, N doesn't change the search direction.
+       N always goes backwards and n always goes forwards.
+
+
+
+
+=================================================================
+=                        CHAPTER 5 RECAP                        =
+=================================================================
+
+ * Type C-s to save position to the jumplist.
+   * Type C-i and C-o to go forward and backward in the jumplist.
+
+ * Type / to search forward in file, and ? to search backwards.
+   * Use n and N to cycle through search matches.
+
+
+
+
+
+
+
+
+
+
+
+
+
+=================================================================
+=                       6.1 JOINING LINES                       =
+=================================================================
+
+ Type J to join together lines in selection.
+
+ 1. Move the cursor to the line below marked -->.
+ 2. Type x three times to select the other lines.
+ 3. Type J to join the lines together.
```

Comment:
Hmm, I didn't know about this, I always used join without selection.

Reply:
True. We could keep the example and change the instructions to press J twice or 2J and then maybe a note saying a you could select the lines and then press J to also cover the joining in selection.

Reply:
That's what I've done.